tests/Unit/ProtoTests/ProtoTest.php

```diff
@@ -68,7 +71,7 @@ private function runProtoTest(
 
     public function testBasic0(): void
     {
-        $this->runProtoTest('Basic/basic.proto');
+        $this->runProtoTest('Basic/basic.proto', null, null, true, MigrationMode::NEW_SURFACE_ONLY);
```

Comment:
I see in `Basic/out` that the `BasicClient` and `BasicGapicClient` were removed!

Super-duper nit to add a comment, and a suggestion to use named parameters for readability!
```suggestion
        // test generating the client with only the new surface (no v1 client, v2 samples)
        $this->runProtoTest('Basic/basic.proto', migrationMode: MigrationMode::NEW_SURFACE_ONLY);
```

Reply:
Oh my gosh named params bless upppppp